flybywiresim/aircraft
# fix(a380x/efis): Fix VV pb indicator not turning on when TRK-FPA is selected

Fixes #9520

## Summary of Changes

Fix VV pb indicator to account for the FPV shown on the PFD when TRK-FPA is selected.

## Screenshots (if necessary)



![image](https://github.com/user-attachments/assets/db7c9039-f02f-475e-ad56-78a460ac36eb)

## References


![image](https://github.com/user-attachments/assets/aa1caf2b-e956-4e42-9a20-0b297fb8e69f)




## Additional context


The testing instructions in #9520 extend into GA mode indications which shouldn't be in the scope of this PR. Will recommend creating a separate issue for fixing GA mode issues if reproduced.


Discord username (if different from GitHub): Heclak

## Testing instructions

1. Check that VV pb is off
2. Press the HDG-V/S/TRK-FPA button to switch to TRK-FPA mode. Check that FPV is shown on PFD and VV pb indicator turns on.
3. Press VV pb to check that VV pb cannot be turned off while TRK-FPA is selected
4. Switch to HDG-V/S mode by pressing the HDG-V/S/TRK-FPA button on the FCU. VV pb indicator should turn off.
5. Press the VV pb to toggle the VV on the PFD. Indicator light on the VV pb should sync with the VV display on the PFD.



## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo**, **flybywire-aircraft-a380-842 (4K)** or **flybywire-aircraft-a380-842 (8K)** download link at the bottom of the page
